src/main/java/com/ldtteam/domumornamentum/block/types/PillarShapeType.java

```diff
@@ -0,0 +1,24 @@
+package com.ldtteam.domumornamentum.block.types;
+
+import net.minecraft.util.StringRepresentable;
+
+public enum PillarShapeType implements StringRepresentable
+{
+    pillar_base ("pillar_base"),
+    pillar_capital ("pillar_capital"),
+    pillar_column ("pillar_column"),
+    full_pillar ("full_pillar");
```

Comment:
Enum name scheme is not followed see the other enums in this project for how to format enum names.